sdk/appconfiguration/app-configuration/test/internal/http.spec.ts

```diff
@@ -44,30 +39,6 @@ describe("http request related tests", function () {
       });
     });
 
```

Comment:
Any reason why these test cases are removed?

Reply:
These tests were used to check the `getUserAgentPrefix` function that was handwritten which is now removed because the generated code handles the user agent string now.

Should I not be doing that?

Reply:
Now that you've asked, the generated code has a different version.
![image](https://user-images.githubusercontent.com/10452642/158487797-55af6ae1-59dd-4c66-a4d2-bc6d5dbe4e9e.png)

How is this supposed to be handled now?
Should I override the useragent option at the convenience layer?

Reply:
I think other packages have `package-version` in swagger readme, which is probably what generated code use. but I could be wrong...